internal/worker/peergrouper/desired.go

```diff
@@ -595,55 +568,20 @@ func (p *peerGroupChanges) updateAddressesFromInternal() error {
 		// address has changed, we enforce the policy of requiring a
 		// configured HA space when there are multiple cloud-local addresses.
 		if !unchanged {
-			multipleAddresses = append(multipleAddresses, id)
-			if err := m.host.SetStatus(getStatusInfo(multiAddressMessage)); err != nil {
-				return errors.Trace(err)
-			}
+			member.Address = selectSingleAddress(addrs)
```

Comment:
I don't think it matters, because at this point we are changing the address anyway. Just do a `sort.Strings` on it before getting index 0.